features/RI007_Stationing-property.feature

```diff
@@ -0,0 +1,39 @@
+@implementer-agreement
+@RI
+Feature: RI007 - Stationing property
+The rule verifies, that station information can be exported in IFC using Pset_Stationing.
+
+  Scenario: Agreement on alignment stationing
+
+    Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcReferent
+      And Its attribute Name
+    Then The value must exist
+
+  Scenario: Agreement on alignment stationing
+
+    Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcReferent
+      And Its attribute IsDefinedBy
+      And Its attribute RelatingPropertyDefinition
+      And Its attribute Name
+    Then The value must exist
+
+  Scenario: Agreement on alignment stationing
+
+    Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcReferent
+      And Its attribute IsDefinedBy
+      And Its attribute RelatingPropertyDefinition
+      And Its attribute HasProperties
+    Then The value of attribute Name must be Station
+
+  Scenario: Agreement on alignment stationing
+
+    Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcReferent
+      And Its attribute IsDefinedBy
+      And Its attribute RelatingPropertyDefinition
+      And Its attribute HasProperties
+      And Its attribute NominalValue
+    Then The value must exist
```

Comment:
These seem like three independent scenarios. I believe the requirement was, however, that all three are valid *simultaneously*. That is, a property set "Pset_Stationing" with a property named "Station" with an existing value.

Imagine a file with some Property.Name="Station" in some property set other than "Pset_Stationing", with a "Pset_Stationing" set present, but without a property named "Station" -> would still make the test pass, but not correspond to the requirement. Perhaps it is a good idea to include such a file in testing as well?